rust/src/proofs/api.rs

```diff
@@ -150,6 +150,24 @@ fn seal_pre_commit_phase1(
     })
 }
 
+/// TODO: document
+#[ffi_export]
+fn generate_sdr(
+    registered_proof: RegisteredSealProof,
+    cache_path: c_slice::Ref<u8>,
```

Comment:
I'm assuming this only generates layers, and we're only interested in the sc-02-data-layer-11.dat.

Separately, does that keep all the layers on disk, or just the last (two) one? Could we add a flag to switch between the two modes if that's easy enough?

Reply:
Currently it's keep all layers around. I'm not sure how much `rust-fil-proofs` currently is into deleting files. Currently SDR is also internally kind of this single big operation API wise. So I'd prefer if it would be OK if we don't remove files while it's running. So would a "delete_all_layers_except_the_last_one_flag" help? Or, if it's done at the end of the whole SDR, would it be done on the Go side then?